requirements.txt

```diff
@@ -27,6 +27,7 @@ netaddr==0.9.0
 Pillow==10.0.1
 psycopg[binary,pool]==3.1.11
 PyYAML==6.0.1
+requests==2.28.1
```

Comment:
The current release is 2.31.0; any reason to stick with the older version?

Reply:
That is what I got doing pip install --upgrade -r requirements.txt so I was assuming it was bound-checked by another library.  Can update it further but I'm not sure if that would have any negative effects.

Reply:
That's odd; I have 2.31.0 installed. Either way, it will get bumped during the release process.